statvar_imports/ndap/nss_education/README.md

```diff
@@ -0,0 +1,17 @@
+source
+: https://ndap.niti.gov.in/catalogue?query=Household+Social+Consumption+-+Education&search=Variables%2CDatasetInfo&domain=ndap
+How to download
```

Comment:
Pls provide specific links to all the datasets to download.

The search result page may change over time and may include new datasets that may not be suitable for this import.